### ISSUE TITLE
💡 [FEATURE] - Add a working Contact Us page 

### ISSUE THREAD

**Issue by Ayushmaanagarwal1211** [jfmartinz/ResourceHub#1397]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

I will add a working contact us page for the website where the message can be directly goes to the email

**Comment by jfmartinz** [comment]
Do you need anything to implement this? Do you need email?

**Comment by Ayushmaanagarwal1211** [comment]
Yes i just wanted the service_id , template_id and public api key of email js it is very simple not more than 5 min i can  send you a video

**Comment by jfmartinz** [comment]
What do you mean by this? Can you conenct with me in [linkedin](https://www.linkedin.com/in/jfmartinz/)

**Comment by Ayushmaanagarwal1211** [comment]
Yes yes actually this will only be implemented by email js so you just go to emailjs.com and create your account and then do log in . Just After that you will get all your credentials . I am attaching a complete Video for that from that you will easily understand .

https://github.com/jfmartinz/ResourceHub/assets/118350936/ee009449-026d-45e4-bb00-a57d47d22573

**Comment by jfmartinz** [comment]
service_xul7il5
template_h4wikn8
hy4H1lNcyJFX5uy-W

**Comment by Ayushmaanagarwal1211** [comment]
> service_xul7il5 template_h4wikn8 hy4H1lNcyJFX5uy-W

Ok thanks i will apply these credentials
